src/main/java/vazkii/botania/client/render/world/SkyblockRenderEvents.java

```diff
@@ -16,13 +16,21 @@
 import vazkii.botania.common.core.handler.ConfigHandler;
 import vazkii.botania.common.world.WorldTypeSkyblock;
 import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import java.util.stream.IntStream;
 
 public final class SkyblockRenderEvents {
 
 	@SubscribeEvent
 	public void onRender(RenderWorldLastEvent event) {
 		World world = Minecraft.getMinecraft().theWorld;
-		if(ConfigHandler.enableFancySkybox && world.provider.dimensionId == 0 && (ConfigHandler.enableFancySkyboxInNormalWorlds || WorldTypeSkyblock.isWorldSkyblock(Minecraft.getMinecraft().theWorld))) {
+
+		boolean renderFancySkybox = ConfigHandler.enableFancySkybox && (
+				(ConfigHandler.enableFancySkyboxInNormalWorlds && (world.provider.dimensionId == 0) ) ||
+				(ConfigHandler.enableFancySkyboxInDimension && (IntStream.of(ConfigHandler.fancySkyboxDimension).anyMatch(x -> x == world.provider.dimensionId))) ||
```

Comment:
You should do a set of the dim ids in the config handler, and just check if the set contains the dim ID instead of doing this Stream at each event